docs/_includes/section-title.html

```diff
@@ -24,6 +24,38 @@ <h2 class="section-title__title">Today’s Deals – All With Free Shipping</h2>
 </div>
     {% endhighlight %}
 
+    <h3 id="section-title-arrow-right">Section Title with CTA Right </h3>
+    <p>This is the new DS Evo style of Section Title. </p>
```

Comment:
Can you check with design whether the old "see all" version is now deprecated and needs removing from our docs?

Reply:
Randy replied saying it has been deprecated. So I will remove it from docs and add a deprecation notice in the code.